try-it.md

```diff
@@ -325,31 +426,45 @@ $ kubectl scale machinedeployment test1-md-0 --replicas=0
 > warning "Warning"
 > control-plane and cluster are very tied together. This means that you are not able to deprovision the control-plane of a cluster and then provision a new one within the same cluster. Therefore, in case you want to deprovision the control-plane you need to **deprovision the cluster** as well and provision both again.
 
-Below, it is shown how the deprovisioning can be executed in a more manual way by just deleting the proper Custom Resources (CR)
+Below, it is shown how the deprovisioning can be executed in a more manual way by just deleting the proper Custom Resources (CR).
+
+The order of deletion is:
+1. Machine objects of the workers
+2. Metal3Machine objects of the workers
+3. Machine objects of the control plane
+4. Metal3Machine objects of the control plane
+5. The cluster object
+
+An additional details is that the `Machine` object `test1-workers-gjcts` is controlled by the the `test1` `MachineDeployment`
+object thus in order to avoid reprovisioning of the `Machine` object the  `MachineDeployment` has to be deleted instead of the `Machine` object in the case of `test1-workers-gjcts`.
 
 ```sh
-$ kubectl delete machine test1-md-0-m87bq -n metal3
-machine.cluster.x-k8s.io "test1-md-0-m87bq" deleted
+$ kubectl delete machinedeployment test1 -n metal3
+machinedeployment.cluster.x-k8s.io "test1" deleted
+
+$ kubectl delete metal3machine test1-workers-gjcts -n metal3
+machine.cluster.x-k8s.io "test1-75678f6485-z928j" deleted
+
+$ kubectl delete machine test1-m77bn -n metal3
+machine.cluster.x-k8s.io "test1-m77bn" deleted
 
-$ kubectl delete machine test1-controlplane-0 -n metal3
-machine.cluster.x-k8s.io "test1-controlplane-0" deleted
+$ kubectl delete metal3machine test1-controlplane-ssd56 -n metal3
+machine.cluster.x-k8s.io "test1-controlplane-ssd56" deleted
```

Comment:
No need for including Metal3Machine deletion step, because when Machine is deleted, corresponding M3Machine will get deleted automatically. That's a reason why in the original document we were showing only deletion of Machine objects, which normally causes deletion of M3Machine, and BMH object deprovisioning.